setup.py

```diff
@@ -21,15 +22,18 @@
     # arguments that distutils doesn't understand
     setuptools_kwargs = {
         'install_requires': [
-            'filechunkio',
             'PyYAML',
-            'simplejson>=2.0.9',
+            'filechunkio',
         ],
         'provides': ['mrjob'],
         'test_suite': 'tests.suite.load_tests',
         'zip_safe': False,  # so that we can bootstrap mrjob
     }
 
+    # ujson is recommended, but requires CPython
+    if platform.python_implementation() == 'CPython':
+        setuptools_kwargs['install_requires'].append('ujson')
```

Comment:
Will do. I've tried to avoid this in general (even `boto` is "optional" in the sense that mrjob can run without it if you don't use EMR), but `ujson` requires a compiler, which might be a showstopper for a lot of people.